### PR TITLE
Add sgro and kroend/kromax params for gas-condensate

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: docs/make_parameter_guideplot.py [hunk 1]
```diff
@@ -27,15 +27,17 @@ def make_gasoil_plot(show=True, krgendanchor="sorg"):
     swl = 0.1
     sgcr = 0.2
     sorg = 0.2
+    sgro = 0.1
```
Comment (vkip) [inline docs/make_parameter_guideplot.py:30]: Need sgro=sgro (for end-point scaling of krosgro(=KROR)

**Review by vkip — COMMENTED** [review]
Need to enforce sgcr=sgro throughout?

File: docs/make_parameter_guideplot.py (hunk 1 above)
Comment (berland) [reply docs/make_parameter_guideplot.py:30]: That would make the plot hard to read (?), as it would not be evident where krosgro is attached.

Comment (vkip) [reply docs/make_parameter_guideplot.py:30]: Could just add '=sgcr' in the annotation (and drop the sgcr ann..)?

**Comment by vkip** [comment]
I would still insist on sgro=sgcr or sgro=0, as other cases are not consistent with the simulator end-point scaling.

**Comment by berland** [comment]
> I would still insist on sgro=sgcr or sgro=0, as other cases are not consistent with the simulator end-point scaling.

sgro=0 will not work in this plot, as it will hide how krosgro works.

sgro=sgcr in this plot will hide which of these endpoints control which curve,  so we will loose information in the plot in both of these cases.

I am not sure that consistency with Eclipse is present in all other plots, as these plots are only made to illustrate how Pyscal works. I suggest such consistency is encouraged in plots coming in PR #283  instead.

**Review by vkip — APPROVED** [review]
Looks good - and now I _think_ we all agree!

**Pull request merged**